demo/right-to-left(rtl).html

```diff
@@ -89,10 +89,10 @@ <h1>RTL Demo</h1>
     };
 
     var widgets = [
-      {x: 0, y: 0, width: 2, height: 2, index: 1},
-      {x: 2, y: 0, width: 4, height: 2, index: 2},
-      {x: 6, y: 0, width: 2, height: 4, index: 3},
-      {x: 1, y: 2, width: 4, height: 2, index: 4}
+      {width: 2, height: 2, index: 1},
```

Comment:
Did you have to remove x,y (same as auto: true). that's the thing adding rtl=true is NOT enough to switch between modes as all CSS and internal code is still (0,0) at top-left and not right side.

while you patch addNode() to flip X around, a lot more code (serialize, and switching column count live) will sort the nodes left->right and so that sortNode() and likely other parts will need to change. OneColumn also uses item sorting to figure out who's first (today it will be top-left).

Reply:
Yeah, as I fiddle with the options there's a bunch of cases I did not consider. The interception logic is not aware of the change in direction, for example.